account_consolidation/tests/test_account_consolidation.py

```diff
@@ -400,57 +396,19 @@ def test_multi_consolidation_(self):
         feb_moves = self.env['account.move.line'].sudo(
             self.consolidation_manager).browse(line_ids).mapped('move_id')
         self.assertTrue(all(feb_moves.mapped('auto_reverse')))
+        self.assertFalse(all(feb_moves.mapped('reverse_date')))
         self.assertFalse(feb_moves.mapped('reverse_entry_id'))
-        # posted move was reversed
-        self.assertTrue(jan_moves[0].reverse_entry_id)
-        self.assertFalse(jan_moves[1].reverse_entry_id)
-        # where created consolidation record and reversal record
-        final_moves_count = self.env['account.move'].search_count([])
-        self.assertEqual(final_moves_count, init_moves_count + 5)
-
-    def test_multi_cron_consolidation_(self):
-        # run consolidation multiple times and check if accounting entries are
-        # created correctly
-        init_moves_count = self.env['account.move'].search_count([])
-        wizard = self.env['account.consolidation.consolidate'].sudo(
-            self.consolidation_manager).create({
-                'month': '01',
-                'target_move': 'all',
-            })
-
-        res = wizard.sudo(self.consolidation_manager).run_consolidation()
-        line_ids = res['domain'][0][2]
-        jan_moves = self.env['account.move.line'].sudo(
-            self.consolidation_manager).browse(line_ids).mapped('move_id')
-        jan_moves.write({'reverse_date': '%s-01-31' % time.strftime('%Y')})
```

Comment:
@grindtildeath  this was for that test of user behavior, when entries can be reversed before consolidation

Reply:
If it's reversed manually by the user, so be it, he must know what he's doing :wink: